packages/actor-query-operation-leftjoin/package.json

```diff
@@ -31,10 +31,10 @@
     "lib/**/*.js"
   ],
   "dependencies": {
-    "@comunica/bus-query-operation": "^2.4.0",
-    "@comunica/bus-rdf-join": "^2.4.0",
-    "@comunica/core": "^2.4.0",
-    "@comunica/types": "^2.4.0",
+    "@comunica/bus-query-operation": "^2.3.0",
```

Comment:
I assume this is because your branch is out-of-sync with master?

Reply:
Related to the yarn.lock merge conflict mentioned below.

Reply:
yep, fixed this.